src/components/settings/TimeZoneSelect.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from "prop-types";
+import i18n from "@dhis2/d2-i18n";
+import { SingleSelectField, SingleSelectOption } from "@dhis2/ui";
+
+const timeZones = Intl.supportedValuesOf("timeZone");
+
+const TimeZoneSelect = ({ timeZone, onChange }) =>
+  timeZones ? (
+    <SingleSelectField
+      filterable
```

Comment:
Should this selectfield have a default value?

Reply:
https://github.com/dhis2/climate-data-app/assets/49228830/c3c5bd8e-2acd-440b-bc9d-37bc73f9b76c

Reply:
![image](https://github.com/dhis2/climate-data-app/assets/49228830/721e97d6-074c-4baa-b032-b899da9824ce)

Reply:
It's a bit hard to find a default value when the time zone returned from the DHIS2 server is not reliable.